src/client/js/components/Admin/UserGroup/UserGroupCreateForm.jsx

```diff
@@ -14,7 +14,9 @@ class UserGroupCreateForm extends React.Component {
       name: '',
     };
 
-    this.xss = window.xss;
+    if (process.browser) {
```

Comment:
これってどういったコードですか...?
もし存在チェックするなら 
```suggestion
    if (process.browser != null) {
```
でお願いします。